apps/alerts/lib/priority.ex

```diff
@@ -16,7 +16,19 @@ defmodule Alerts.Priority do
     :low
   end
 
-  def priority(%{effect: :delay}, _) do
+  def priority(
+        %{effect: :delay, severity: severity, informed_entity: informed_entities},
```

Comment:
question: don't all delay alerts have a severity and informed_entity? If so wouldn't `def priority(%{effect: :delay}, _params)` be unreachable?

Reply:
I was a little unsure if we were guaranteed to have an informed entity here or not. If we do always have one, then this would be unreachable. I can go ahead and remove it if that's the case.

Reply:
This one only applies to delay alerts with a severity under 5 (line 23), the rest will all still use `def priority(%{effect: :delay}, _params)`.